exec/csnex/Opcode.cs

```diff
@@ -0,0 +1,111 @@
+﻿namespace csnex
+{
+    public class NeonOpcode
```

Comment:
What is the purpose of the class wrapper here? Can't you just declare a constant? Let me know if this is a C# thing I don't understand.

Reply:
Yea.  Namespaces cannot contain fields or methods.  They have to be in a class.

Reply:
Moved BYTECODE_VERSION to Bytecode.cs instead.